packages/community-jazz-vue/src/tests/useAcceptInvite.test.ts

```diff
@@ -0,0 +1,55 @@
+// @vitest-environment happy-dom
+
+import { CoMap, Group, type ID, coField } from "jazz-tools";
+import { describe, expect, it } from "vitest";
+import { createInviteLink, useAcceptInvite } from "../index.js";
+import { createJazzTestAccount, linkAccounts } from "../testing.js";
+import { waitFor, withJazzTestSetup } from "./testUtils.js";
+
+describe("useAcceptInvite", () => {
+  it("should accept the invite", async () => {
+    class TestMap extends CoMap {
+      value = coField.string;
+    }
```

Comment:
This should be ported to Zod based schemas

```suggestion
    const TestMap = co.map({
      value: z.string()
    })
```

Reply:
This has been updated across all tests.